gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineTesting.java

```diff
@@ -46,6 +46,12 @@ public void apply(Project project) {
                 // Never cache test tasks, until we work out the correct inputs for ETE / integration tests
                 task.getOutputs().cacheIf(t -> false);
             }
+
+            // repos that use 'snapshot' style testing should all use one convenient task to refresh the snapshots,
+            // ./gradlew test -Drecreate=true
+            String shouldRecreate = System.getProperty("recreate", "false");
+            task.systemProperty("recreate", shouldRecreate);
+            task.getInputs().property("recreate", shouldRecreate);
```

Comment:
Should be be defensive and disable caching when shouldRecreate=true?

Reply:
We may want to require/force boolean values here, possibly `Boolean.getBoolean("recreate")` and use the string value for the sysprop+input?